subgraph/origin/schema.graphql

```diff
@@ -0,0 +1,33 @@
+type DeclaredDepositIntent @entity {
+  id: ID!
+  tokenAddress: Bytes! # address
+  amount: BigInt! # uint256
+  nonce: BigInt!  # uint256
+  beneficiary: Bytes! # address
+  feeGasPrice: BigInt! # uint256
+  feeGasLimit: BigInt! # uint256
+  depositor: Bytes! # address
+  blockNumber: BigInt! # uint256
+  blockHash: Bytes! # bytes32
+  contractAddress: Bytes! # address
+  uts: BigInt! # uint256
+}
+
+type ConfirmedWithdrawIntent @entity {
+  id: ID!
+  messageHash: Bytes! # bytes32
+  blockNumber: BigInt! # uint256
+  blockHash: Bytes! # bytes32
+  contractAddress: Bytes! # address
+  uts: BigInt! # uint256
+}
+
+type ProvenGateway @entity {
+  id: ID!
+  provenBlockNumber: BigInt! # uint256
+  gatewayAddress: Bytes! # address
```

Comment:
```suggestion
  remoteGatewayAddress: Bytes! # address
```

Reply:
I think we can align ordering same as contract event.
provenBlockNumber can come after remoteGatewayAddress.
What do you think?

Reply:
Yah, we can align ordering same as contract. Changed.